config/schemas/settings.json

```diff
@@ -474,7 +474,7 @@
 				},
 				"defaultRepositoryURL" : {
 					"type" : "string",
-					"default" : "https://raw.githubusercontent.com/vcmi/vcmi-mods-repository/develop/vcmi-1.3.json",
+					"default" : "https://raw.githubusercontent.com/vcmi/vcmi-mods-repository/develop/vcmi-1.4.json",
```

Comment:
Please adjust also https://github.com/vcmi/vcmi/blob/3b06abd0d7d1534a507ebae111ca7003b8d5d189/android/vcmi-app/src/main/java/eu/vcmi/vcmi/ActivityMods.java#L53 if you change the repo.